### ISSUE TITLE
Markdown preview fails to load on any project

### ISSUE THREAD

**Issue by thejavascriptman** [microsoft/vscode#97503]
Version: 1.46.0-insider
Commit: 2b472d1e4aaaecdda5ca96289869f90ae82de3a1
Date: 2020-05-11T10:00:38.669Z
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Linux x64 5.4.38-1-MANJARO

Steps to Reproduce:

1. Launch vs code
2. Open any directory that contains a `.md` file
3. Approximately 1 out of 5 times the markdown preview will fail to load until you completely close and reopen vs code

Does this issue occur when all extensions are disabled?: Yes, also happens on a clean install

Screenshot:

![2020-05-11_22-17](https://user-images.githubusercontent.com/46071365/81566725-40f05c80-93d6-11ea-832b-635c0ea53210.png)

**Comment by thejavascriptman** [comment]
I was able to reproduce the issue consistently by using the version reported in the first post, as well as the version mentioned below:

Version: 1.46.0-insider
Commit: c7a37c4400bb4c1b92eeeaea1bcacc9ec3789ef3
Date: 2020-06-04T08:30:33.216Z
Electron: 7.3.1
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Linux x64 5.4.43-1-MANJARO

Steps to reproduce:
* Close all vs code windows
* Launch a new vs code window
* Open any project (preferably an empty folder)
* Open or create a markdown file and make sure the markdown has some contents
* Open the markdown preview 
* Exit vs code without closing the markdown preview
* Reopen vs code

**Comment by mjbvz** [comment]
Please test this gain with the latest VS Code insiders build (1.47). I'm not able to reproduce this issue following your steps, even after trying 10 times or so

**Comment by thejavascriptman** [comment]
Issue does not happen on latest insider 1.47 (b9d7ec9ff367a8e59fe4cc2abee605a55be180e8)

However I'm getting a different error when running latest master (9c142ea3831a60fa62ee62cbcaa6a10e4b2a614d) after following similar steps as before

Steps:
* Launch code by running `./scripts/code.sh`
* Open any project (preferably an empty folder)
* Open or create a markdown file and make sure the markdown has some contents
* Open the markdown preview
* Exit vs code without closing the markdown preview
* Reopen vs code by running `./scripts/code.sh`

Error:
```
[9802:0620/091201.704887:INFO:CONSOLE(144)] "%c  ERR color: #f33 Cannot read property 'onDidChange' of undefined: TypeError: Cannot read property 'onDidChange' of undefined
    at EditorPart.get onDidChange [as onDidChange] (file:///home/user/Desktop/vscode/out/vs/workbench/browser/parts/editor/editorPart.js:582:80)
    at new LeafNode (file:///home/user/Desktop/vscode/out/vs/base/browser/ui/grid/gridview.js:415:65)
    at GridView._deserializeNode (file:///home/user/Desktop/vscode/out/vs/base/browser/ui/grid/gridview.js:831:26)
    at file:///home/user/Desktop/vscode/out/vs/base/browser/ui/grid/gridview.js:824:36
    at Array.map (<anonymous>)
    at GridView._deserializeNode (file:///home/user/Desktop/vscode/out/vs/base/browser/ui/grid/gridview.js:822:53)
    at file:///home/user/Desktop/vscode/out/vs/base/browser/ui/grid/gridview.js:824:36
    at Array.map (<anonymous>)
    at GridView._deserializeNode (file:///home/user/Desktop/vscode/out/vs/base/browser/ui/grid/gridview.js:822:53)
    at file:///home/user/Desktop/vscode/out/vs/base/browser/ui/grid/gridview.js:824:36
    at Array.map (<anonymous>)
    at GridView._deserializeNode (file:///home/user/Desktop/vscode/out/vs/base/browser/ui/grid/gridview.js:822:53)
    at GridView._deserialize (file:///home/user/Desktop/vscode/out/vs/base/browser/ui/grid/gridview.js:816:30)
    at Function.deserialize (file:///home/user/Desktop/vscode/out/vs/base/browser/ui/grid/gridview.js:812:20)
    at Function.deserialize (file:///home/user/Desktop/vscode/out/vs/base/browser/ui/grid/grid.js:424:50)
    at Workbench.createWorkbenchLayout (file:///home/user/Desktop/vscode/out/vs/workbench/browser/layout.js:985:59)
    at file:///home/user/Desktop/vscode/out/vs/workbench/browser/workbench.js:78:26
    at InstantiationService.invokeFunction (file:///home/user/Desktop/vscode/out/vs/platform/instantiation/common/instantiationService.js:43:24)
    at Workbench.startup (file:///home/user/Desktop/vscode/out/vs/workbench/browser/workbench.js:61:38)
    at DesktopMain.open (file:///home/user/Desktop/vscode/out/vs/workbench/electron-browser/desktop.main.js:61:52)", source: file:///home/user/Desktop/vscode/out/vs/platform/log/common/log.js (144)
```

**Comment by mjbvz** [comment]
Thanks @thejavascriptman for following up. I suspect this was fixed by 2c53ad120d952fd1ea6e23d3b8e0c3276faf529d

The new error you are seeing is #100783 which we are investigating
